std/encoding/tests/gen_signature/definition.go

```diff
@@ -24,6 +24,9 @@ type T1 struct {
 
 	//+field:procedureArgument:enc.Wire
 	sigCovered enc.PlaceHolder
+
+	//+field:natural:optional
+	H3 optional.Optional[uint64] `tlv:"0x06"`
```

Comment:
What is H3?

Reply:
Test case to ensure we handle fields outside the sigCovered part correctly, which appear after the signature.